### ISSUE TITLE
[Bug] translate pipe parameters still don't work when there's a space inbetween (in certain cases)

### ISSUE THREAD

**Issue by mgrilec** [transifex/transifex-javascript#183]
**Current Behavior**
When formatting HTML with prettier, translate pipes can come out with spaces and in multiple lines. This behaviour is not configurable. Running tx-js push on those files fails to parse the key and tags.

**Expected Behavior**
The key and tags should be parsed.

**Steps to Reproduce**

`npm i @transifex/cli`

```test.html
<mat-tab
      label="{{
        'My str'
          | translate
            : {
                _key: 'my-key',
                _tags: 'my-tag'
              }
      }}"
    >
```

`npx txjs-cli push test.html --dry-run --verbose`

```
Parsing all files to detect translatable content...
⸨████████████████████████████████████████⸩ 1/1 test.html
✓ Processed 1 file(s) and found 1 translatable phrases.
✓ Content detected in 1 file(s).
test.html
  └─ My str
    └─ occurrences: ["test.html"]
```

Notice that the key and tags weren't parsed.

**Possible Solution**
???

**Possible Implementation**

Looks like the following format works, so it might be an issue with the surrounding ".

```test2.html
<p>{{
      'My other str'
        | translate
          : {
              _key: 'my-other-key',
              _tags: 'my-other-tag'
            }
    }}</p>
```

**Comment by pablotransifex** [comment]
@mgrilec Thank you for the feedback, we'll look into it. and let you know ASAP.

**Comment by pablotransifex** [comment]
@mgrilec We've released the version 5.1.1. Please feel free to check if the issue is fixed for you.

**Comment by mgrilec** [comment]
Looks like it works. Thanks!
